HLTriggerOffline/Common/python/HLTValidationHarvest_cff.py

```diff
@@ -42,11 +42,11 @@
 # Temporary Phase-2 configuration
 # Exclude everything except JetMET for now
 from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
-phase2_common.toReplaceWith(hltpostvalidation, hltpostvalidation.copyAndExclude([HLTMuonPostVal,
+phase2_common.toReplaceWith(hltpostvalidation, hltpostvalidation.copyAndExclude([#HLTMuonPostVal,
                                                                                  HLTTauPostVal,
                                                                                  EgammaPostVal,
                                                                                  postProcessorHLTgsfTrackingSequence,
-                                                                                 postProcessorHLTmuonTrackingSequence,
+                                                                                 #postProcessorHLTmuonTrackingSequence,
```

Comment:
why keeping it commented?

Reply:
In both cases, I've followed the same approach as the other validation sequences (that I assumed were commented each time one got integrated). There would be no problem outright removing them

Reply:
I'd recommend to remove them, given this PR generally addresses adding back the muon object DQM/ validation sequences (and also to remove unncessary clutter).